foundation-ui/services/MountBundle.js

```diff
@@ -0,0 +1,147 @@
+import React, {PropTypes} from 'react';
```

Comment:
The problem with this structure is that we need to instantiate the `Hooks` to be used in both the service and component. To make it completely isolated we chose do to it in one file.
What I did was to then have a separate file for the Hooks instantiation (which we will need for every service/component pair) that the two files could include.

It will then look something like this:
* foundation
	* mount
		* hooks.js
		* MountService.js
		* MountComponent.js
	* auth
		* hooks.js
		* AuthService.js
		* AuthComponent.js
	* ...
I removed the `main.js` or `index.js` to not cause confusion of what you will get, but we could include it to expose the service and component.

One last thing: We don't postfix with `Component` or `Service` anywhere else, which I think would be good here to differentiate, but this looks kind of weird to me:
```js
<MountComponent id="foo">
  <h1>bar</h1>
</MountComponent>
```

However, naming it `Mount` we could make that be exposed in the `index.js`, so you could do this:
`import Mount from 'foundation-ui/Mount';`
or this:
`import Mount from 'foundation-ui/Mount/Mount';`
`import MountService from 'foundation-ui/Mount/MountService';`